git/git.go

```diff
@@ -76,11 +76,13 @@ type Commit struct {
 	Title string
 }
 
-func Commits(baseRef, headRef string) ([]*Commit, error) {
+func Commits(remote, baseRef, headRef string) ([]*Commit, error) {
```

Comment:
The `Comits()` method is not used anywhere thats why I thought to modify the signature and implementation. From the title of the method I think It can be used in other places in the future. In that case the whole thing can be done in the `computeDefaults` method. Thanks!